test/Contract/DelegateServer.purs

```diff
@@ -0,0 +1,301 @@
+import Control.Parallel (parTraverse_)
+import Data.Array (foldRecM, last, replicate, singleton, snoc) as Array
+import Data.Time.Duration (Seconds(Seconds))
+import DelegateServer.Contract.PlaceBid
+  ( PlaceBidL2ContractError(PlaceBidL2_Error_InvalidBidStateTransition)
+  )
+import DelegateServer.Handlers.MoveBid
+  ( MoveBidSuccess(MoveBidSuccess_SentInitHeadRequest, MoveBidSuccess_CommittedStandingBid)
+  )
+import DelegateServer.Handlers.PlaceBid
+  ( PlaceBidError(PlaceBidError_ContractError)
+  , PlaceBidSuccess(PlaceBidSuccess_SubmittedTransaction)
+  )
+import DelegateServer.Types.AppExitReason (AppExitReason(AppExitReason_HeadFinalized))
+import DelegateServer.Types.HydraHeadStatus
+  ( HydraHeadStatus
+      ( HeadStatus_Idle
+      , HeadStatus_Initializing
+      , HeadStatus_Open
+      , HeadStatus_Closed
+      )
+  )
+import DelegateServer.Types.ServerResponse
+  ( ServerResponse(ServerResponseSuccess, ServerResponseError)
+  )
+import HydraAuctionOffchain.Contract (discoverBidders)
+import HydraAuctionOffchain.Contract.QueryUtxo (queryStandingBidUtxo)
+import HydraAuctionOffchain.Contract.Types
+  ( AuctionInfoExtended
+  , BidTerms(BidTerms)
+  , BidderInfo(BidderInfo)
+  , StandingBidState(StandingBidState)
+  , bidderSignatureMessage
+  )
+import HydraAuctionOffchain.Helpers (mkPosixTimeUnsafe, waitSeconds)
+import HydraAuctionOffchain.Wallet (signMessage)
+import JS.BigInt (BigInt, fromInt)
+import Mote (group, skip, test)
+import Partial.Unsafe (unsafePartial)
+import Test.Contract.AnnounceAuction (AuctionTermsMutator, announceAuctionFix)
+import Test.Contract.AuthorizeBidders (authorizeBidders)
+import Test.Contract.EnterAuction (enterAuction, genValidBidderDeposit)
+import Test.Contract.Fixtures (minBidIncrementFixture, startingBidFixture)
+import Test.Contract.PlaceBid (discoverSellerSignature)
+import Test.Contract.StartBidding (startBidding)
+import Test.DelegateServer.Cluster (TestAppHandle, withWallets')
+import Test.Helpers (defDistribution, randomElem, untilM, waitUntil)
+import Test.QuickCheck.Gen (Gen, chooseInt, randomSampleOne)
+import Test.Spec.Assertions (shouldEqual, shouldReturn, shouldSatisfy)
+
+suite :: TestPlanM ContractTest Unit
+suite =
+  group "delegate-server" do
+    test "delegates set announced auction as active auction" do
+      withWallets' defDistribution
+        \seller delegates withDelegateServerCluster -> do
+          { txHash, auctionInfo } <-
+            withKeyWallet seller $
+              announceAuctionFix (_ { delegates = delegates })
+          awaitTxConfirmed txHash
+
+          withDelegateServerCluster auctionInfo \appHandle ->
+            shouldEqual auctionInfo =<<
+              untilJust do
+                waitSeconds one
+                appHandle.getActiveAuction
+
+    -- NOTE: All nodes in the hydra head fail to submit an InitTx
+    -- transaction with NotEnoughFuel error when raiseExUnitsToMax
+    -- parameter of PlutipConfig is set to true.
```

Comment:
@errfrom could you please elaborate on this?

Reply:
This is just an observation, and I didn't invest any time to investigate it because it's not critical for testing